src/components/data/eventsAndFactsData.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+      "id": 1,
+      "event": "Sudden uptick in immigration from Europe",
+      "fact": "in 1914 Duncan McDuffie eal estate developer Duncan McDuffie was one of the early proponents of single-family zoning in this neighborhood of Berkeley to prevent a dance hall owned by a Black resident from moving into houses he was trying to sell. He worried that families of color moving into the neighborhood would decrease the desirability of the neighborhood and decrease property values. By advocating for single-family zoning, McDuffie and other developers at the time were attempting to price out social groups whom they deemed to be less desirable for the neighborhood.[1] This makes single-family zoning one of many exclusionary zoning policies intended to limit who was able to afford living in a certain neighborhood. The goal of limiting certain neighborhoods to only single-family homes meant that only families who could afford to buy an entire house could live in the neighborhood. There was not the option to subdivide housing so that families who could not afford to buy the whole property could live in smaller units.",
+      "links": [
+        "https://en.wikipedia.org/wiki/Duncan_McDuffie",
+        "https://en.wikipedia.org/wiki/Single-family_zoning#cite_note-KQED-1",
+        "https://en.wikipedia.org/wiki/Single-family_zoning#cite_note-B-side_1-13"
+      ]
+  },
```

Comment:
```
  {
      "id": 1,
      {
        "question": "What do you choose?",
        "image": "url-link/relative-link-in-assets",
        "options": [{
            "choiceId": 1,
            "option": "Choose and move to 2",
            "densityEffect": 1,
            "infrastractureEffect": 1,
            "nextId": 2
          },
          {
            "choiceId": 2,
            "option": "Choose and move to 3",
            "densityEffect": 1,
            "infrastractureEffect": 1,
            "nextId": 2
          }
        ]
      },
      "event": "Sudden uptick in immigration from Europe",
      "fact": "in 1914 Duncan McDuffie eal estate developer Duncan McDuffie was one of the early proponents of single-family zoning in this neighborhood of Berkeley to prevent a dance hall owned by a Black resident from moving into houses he was trying to sell. He worried that families of color moving into the neighborhood would decrease the desirability of the neighborhood and decrease property values. By advocating for single-family zoning, McDuffie and other developers at the time were attempting to price out social groups whom they deemed to be less desirable for the neighborhood.[1] This makes single-family zoning one of many exclusionary zoning policies intended to limit who was able to afford living in a certain neighborhood. The goal of limiting certain neighborhoods to only single-family homes meant that only families who could afford to buy an entire house could live in the neighborhood. There was not the option to subdivide housing so that families who could not afford to buy the whole property could live in smaller units.",
      "links": [
        "https://en.wikipedia.org/wiki/Duncan_McDuffie",
        "https://en.wikipedia.org/wiki/Single-family_zoning#cite_note-KQED-1",
        "https://en.wikipedia.org/wiki/Single-family_zoning#cite_note-B-side_1-13"
      ]
  },
```

Reply:
is this ok? since we are wrapping the question as object
[
  {
    "id": 1,
    "question": {
      "text": "It's 1914. You are a 35 yr old developer named Muffin McMuffy in Merkely, CA you have major influence over local politics because you've greased a few palms in your day...",
      "image": "q1_house_1.png",
      "options": [{
        "choiceid": 1,
        "option": "Choose and move to 2",
        "nextId": 2
        },
        {
          "qid": 2,
          "option": "Choose and move to 3",
          "nextId": 3
        }
      ]
    },
    "event": "Sudden uptick in immigration from Europe",
    "fact": "in 1914 Duncan McDuffie eal estate developer Duncan McDuffie was one of the early proponents of single-family zoning in this neighborhood of Berkeley to prevent a dance hall owned by a Black resident from moving into houses he was trying to sell. He worried that families of color moving into the neighborhood would decrease the desirability of the neighborhood and decrease property values. By advocating for single-family zoning, McDuffie and other developers at the time were attempting to price out social groups whom they deemed to be less desirable for the neighborhood.[1] This makes single-family zoning one of many exclusionary zoning policies intended to limit who was able to afford living in a certain neighborhood. The goal of limiting certain neighborhoods to only single-family homes meant that only families who could afford to buy an entire house could live in the neighborhood. There was not the option to subdivide housing so that families who could not afford to buy the whole property could live in smaller units.",
  }, 
  {..}
]

Reply:
{
  "id": 1,
  "question": {
    "text": "It's 1914. You are a 35 yr old developer named Muffin McMuffy in Merkely, CA you have major influence over local politics because you've greased a few palms in your day...",
    "image": "q1_house_1.png",
    "options": [
      {
        "choiceid": 1,
        "option": "Choose and move to 2",
        "nextId": 2
      },
      {
        "qid": 2,
        "option": "Choose and move to 3",
        "nextId": 3
      }
    ]
  },
  "event": "Sudden uptick in immigration from Europe",
  "fact": "in 1914 Duncan McDuffie eal estate developer Duncan McDuffie was one of the early proponents of single-family zoning in this neighborhood of Berkeley to prevent a dance hall owned by a Black resident from moving into houses he was trying to sell. He worried that families of color moving into the neighborhood would decrease the desirability of the neighborhood and decrease property values. By advocating for single-family zoning, McDuffie and other developers at the time were attempting to price out social groups whom they deemed to be less desirable for the neighborhood.[1] This makes single-family zoning one of many exclusionary zoning policies intended to limit who was able to afford living in a certain neighborhood. The goal of limiting certain neighborhoods to only single-family homes meant that only families who could afford to buy an entire house could live in the neighborhood. There was not the option to subdivide housing so that families who could not afford to buy the whole property could live in smaller units."
}